src/main/resources/application.yaml

```diff
@@ -20,3 +20,6 @@ spring:
 server:
   servlet:
     context-path: /api
+
+file:
+  path: "src\\main\\resources\\files\\"
```

Comment:
Suggestion: leave only `files`, or even `cv`
1. We can't leave these separators since it's invalid path for other operation system (like Linux).
2. I think it's good to save files on the root level.

 @viktarkolbik @SergeyKoval What do you think about path for file storage?

Reply:
done